docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -471,24 +480,37 @@ There's also a `clever playbook <https://github.com/ansible/ansible-examples/blo
 
 .. _tips_and_tricks:
```

Comment:
I might update the anchor here to match the new title below (I like the new title better - it's more informative).